app/editor/src/features/content/form/ContentStoryForm.tsx

```diff
@@ -239,6 +239,25 @@ export const ContentStoryForm: React.FC<IContentStoryFormProps> = ({
               setEffort={setEffort}
               userId={userId!}
             />
+            <Modal
+              hide={toggle}
+              isShowing={isShowing}
+              headerText="Prep Time Log"
+              body={
+                <TimeLogTable
+                  setTotalEffort={setEffort}
+                  totalEffort={effort}
+                  data={values.timeTrackings}
+                />
+              }
+              customButtons={
+                <Button variant={ButtonVariant.secondary} onClick={toggle}>
+                  Close
+                </Button>
+              }
+            />
+          </Show>
+          <Show visible={contentType === ContentTypeName.Snippet || showExpandModal}>
```

Comment:
I don't think `showExpandModal` is relevant here.

Reply:
The Modal visibility should be controlled the way modals are controlled normally.  @jdtoombs may be able to help.

Reply:
Code adjusted, thanks